go.mod

```diff
@@ -28,3 +28,6 @@ require (
 )
 
 go 1.13
+
+// Temporary fix, waiting for https://github.com/go-git/go-git-fixtures/pull/6.
+replace github.com/go-git/go-git-fixtures/v4 => github.com/ZauberNerd/go-git-fixtures/v4 v4.3.2-0.20221206142533-16c45f9dbd0a
```

Comment:
The other PR has been merged, please remove the `replace` and update the `go-git-fixtures` in the `require` section to:

```
github.com/go-git/go-git-fixtures/v4 v4.3.2-0.20230305113008-0c11038e723f
```